apps/www/_blog/2022-08-15-launch-week-five-cli-v1-and-admin-api-beta.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Launch Week 5: CLI v1 and Admin API beta'
+description: We are moving Supabase CLI v1 out of beta, and releasing Admin API beta.
+author: soedirgo, qiao
+image: lw5-cli/thumbnail.jpg
+thumb: lw5-cli/thumbnail.jpg
+tags:
+  - launch-week
+date: '2022-08-15'
+toc_depth: 3
+---
+
+# LW5: CLI v1 and Admin API beta
+
+Today we are moving the Supabase CLI v1 out of beta. The Supabase CLI is capable of managing database migrations and generating TypeScript types. Follow these [install instructions](https://github.com/supabase/cli#getting-started) to get started.
+
+In addition, we are also announcing the Admin API beta. The Admin API is a REST API that allows you to manage organizations, projects, Edge Functions, and more. You can read the [docs](https://www.notion.so/Docs-ca2dbdd3f4954da49132d44bead83f1e) or interact with the Admin API from the Supabase CLI v1.
+
+## Manage organizations, projects, Edge Functions, and more
+
+If you’ve used [Supabase Edge Functions](https://supabase.com/blog/2022/03/31/supabase-edge-functions#quickstart), you’ve used the Supabase CLI. We’re extending it to let you manage organizations and projects.
+
+```bash
+supabase login
+supabase orgs list
+supabase projects create my-project --org-id cool-green-pqdr0qc --db-password ******** --region us-east-1
+```
+
+Once created, link your Supabase project locally to begin development.
```

Comment:
What does "created" refer to here?

Reply:
it refers to a new supabase projected. rephrased